packages/components/src/components/input/input.stories.ts

```diff
@@ -69,731 +35,219 @@ export const Default = {
+  `
 };
 
 /**
- * Add the `password-toggle` attribute to add a toggle button that will show the password when activated. Only works with `type="password"`.
+ * Use the `disabled` attribute to disable the input.
  */
 
-export const TogglePassword = {
-  name: 'Toggle Password',
-  parameters: {
-    controls: {
-      exclude: ['password-toggle']
-    }
-  },
-  args: overrideArgs([{ type: 'attribute', name: 'type', value: 'password' }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args,
-          constants: [
-            { type: 'attribute', name: 'password-toggle', value: true },
-            { type: 'attribute', name: 'label', value: 'Password' }
-          ]
-        })}
-      </div>
-    `;
-  }
+export const Disabled = {
+  render: () =>
+    html`<div class="w-[250px]">
+      <sd-input label="Label" placeholder="Input text disabled" disabled></sd-input>
+    </div>`
 };
 
 /**
- * Use the disabled attribute to disable an input. All interaction is disabled and no events will be fired.
+ * Use the `read-only` attribute to render an input as read only.
  */
-
-export const Disabled = {
-  parameters: {
-    controls: {
-      exclude: ['disabled']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          constants: [{ type: 'attribute', name: 'disabled', value: true }],
-          args
-        })}
-      </div>
-    `;
-  }
+export const ReadOnly = {
+  render: () =>
+    html`<div class="w-[250px]">
+      <sd-input label="Label" placeholder="Read only example" readonly></sd-input>
+    </div>`
 };
 
 /**
- * Use the readonly attribute to render an input as readonly.  Interaction is enabled, but the input cannot be edited.  Events will be fired.
+ * Use the `help-text` attribute to add a descriptive “help text”. For help texts that contain HTML, use the `help-text` slot instead.
  */
 
-export const Readonly = {
-  parameters: {
-    controls: {
-      exclude: ['readonly']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
+export const HelpText = {
+  render: () => {
     return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          constants: [{ type: 'attribute', name: 'readonly', value: true }],
-          args
-        })}
+      <div class="flex flex-rows gap-12">
+        <sd-input label="Label" help-text="Help text"></sd-input>
+        <sd-input label="Label">
+          <div slot="help-text" class="text-lg">Help text</div>
+        </sd-input>
```

Comment:
One Input should be enough here (size lg).

![image](https://github.com/user-attachments/assets/88c8519d-bfa6-4069-9ea8-22b48b3ebe0f)

Reply:
Same as https://github.com/solid-design-system/solid/pull/1322#discussion_r1738289397